modules/http4s/README.md

```diff
@@ -0,0 +1,35 @@
+
+# Http4s module for PureConfig
```

Comment:
I wonder if we should add something here saying that only `Uri` is supported for now. People may come here expecting readers for many HTTP model classes and find out that it's not the case. I'm ok with leaving this as it is, though :)

Reply:
Well it wouldn't hurt to do that, so I'll do it :)